### PR TITLE
search 7.4 | whats new in search | LRDOCS-9941

### PR DESCRIPTION
**Pull request by rbohl**
https://issues.liferay.com/browse/LRDOCS-9941
cc @lipusz

### CODE REVIEW

File: docs/dxp/latest/en/using-search/getting-started/whats-new-in-search-for-74.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+## Search Tuning
+
+> **DXP Subscribers**
+
+### Search Tuning Data is Stored in the Database 
+
+Beginning in DXP Liferay 7.4, the Result Rankings and Synonym Sets data is stored in the Liferay database, and a re-index action from the Search Administration panel can be used to populate each application's index with the proper data. In prior versions the indexes were used as primary data storage for these applications.
+
+To learn about how this impacts the upgrade process, read [Upgrading Search Infrastructure](../installing-and-upgrading-a-search-engine/elasticsearch/upgrading-elasticsearch/upgrading-search-infrastructure.md) and [Backing Up Elasticsearch](../installing-and-upgrading-a-search-engine/elasticsearch/upgrading-elasticsearch/backing-up-elasticsearch.md).
```
Comment (jrhoun) [inline docs/dxp/latest/en/using-search/getting-started/whats-new-in-search-for-74.md:52]: > [Upgrading Search Infrastructure](../installing-and-upgrading-a-search-engine/elasticsearch/upgrading-elasticsearch/upgrading-search-infrastructure.md)

^ This link is broken, what would you want it to point to? `upgrading-to-elasticsearch-7.md`? Or something else?

File: docs/dxp/latest/en/using-search/getting-started/whats-new-in-search-for-74.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+## Upgrading to DXP 7.4
+
+Upgrading from a previous DXP version to 7.4 requires no special considerations for most setups. If using Liferay Enterprise Search or Search Tuning, however, make sure you consider these points: 
+
+1. The Search Tuning applications are now backed by database tables, whereas previous versions used Elasticsearch indexes for their data storage. See [Upgrading Search Infrastructure](../installing-and-upgrading-a-search-engine/elasticsearch/upgrading-elasticsearch/upgrading-search-infrastructure.md) for more information.
+1. The Liferay Enterprise Search applications are bundled with Liferay DXP. When upgrading to 7.4 no additional installation steps are required.
+
+## Related Topics
+
+* [Upgrading Search Infrastructure](../installing-and-upgrading-a-search-engine/elasticsearch/upgrading-elasticsearch/upgrading-search-infrastructure.md)
```
Comment (jrhoun) [inline docs/dxp/latest/en/using-search/getting-started/whats-new-in-search-for-74.md:75]: Same question as https://github.com/jrhoun/liferay-learn/pull/1279/files#r747875572

File: docs/dxp/latest/en/using-search/getting-started/whats-new-in-search-for-74.md (hunk 1 above)
Comment (rbohl) [reply docs/dxp/latest/en/using-search/getting-started/whats-new-in-search-for-74.md:52]: Yeah this appears in a few places. It should :crossed_fingers: resolve when Rich merges https://github.com/sez11a/liferay-learn/pull/641

File: docs/dxp/latest/en/using-search/getting-started/whats-new-in-search-for-74.md (hunk 2 above)
Comment (rbohl) [reply docs/dxp/latest/en/using-search/getting-started/whats-new-in-search-for-74.md:75]: Same answer as https://github.com/jrhoun/liferay-learn/pull/1279#discussion_r747877310 :grin:

**Comment by jrhoun** [comment]
Merged. Thanks. I made a few changes here; 77a99c08741deaaac667f8810fd05e5d207fa2c0

The most notable changes are:
* Updated landing card to point to the new article
* Moved 7.4 article higher in the Getting Started section
* Migrated a couple .rst pages to .md so that links to them will resolve

**Pull request closed without merging**